README.md

```diff
@@ -15,122 +15,136 @@ npm install http-status-codes --save
+| 418  | IM_A_TEAPOT                     | I'm a teapot                    |
+| 419  | INSUFFICIENT_SPACE_ON_RESOURCE  | Insufficient Space on Resource  |
+| 420  | METHOD_FAILURE                  | Method Failure                  |
+| 422  | UNPROCESSABLE_ENTITY            | Unprocessable Entity            |
+| 423  | LOCKED                          | Locked                          |
+| 424  | FAILED_DEPENDENCY               | Failed Dependency               |
+| 428  | PRECONDITION_REQUIRED           | Precondition Required           |
+| 429  | TOO_MANY_REQUESTS               | Too Many Requests               |
+| 431  | REQUEST_HEADER_FIELDS_TOO_LARGE | Request Header Fields Too Large |
+| 500  | INTERNAL_SERVER_ERROR           | Server Error                    |
+| 501  | NOT_IMPLEMENTED                 | Not Implemented                 |
+| 502  | BAD_GATEWAY                     | Bad Gateway                     |
+| 503  | SERVICE_UNAVAILABLE             | Service Unavailable             |
+| 504  | GATEWAY_TIMEOUT                 | Gateway Timeout                 |
+| 505  | HTTP_VERSION_NOT_SUPPORTED      | HTTP Version Not Supported      |
+| 507  | INSUFFICIENT_STORAGE            | Insufficient Storage            |
+| 511  | NETWORK_AUTHENTICATION_REQUIRED | Network Authentication Required |
+
+## Migrating from v1.x.x
+
+v2 is backwards compatible with v1, but if you are migrating from http-status-codes v1 to v2, there are a couple of changes that are recommended.
+
+### getStatusText renamed getReasonPhrase
+
+To fix this, simply rename `getStatusText()` to `getReasonPhrase()`. The function is otherwise the same as it was before.
+
+**bad** ❌
 
-```typescript
-import * as HttpStatus from 'http-status-codes'
+```
+getStatusText(200);
+```
 
-response
-	.status(HttpStatus.OK)
-	.send('ok')
+**good** ✅
 
-response
-	.status(HttpStatus.INTERNAL_SERVER_ERROR)
-	.send({
-		error: HttpStatus.getStatusText(HttpStatus.INTERNAL_SERVER_ERROR)
-	})
+```
+getReasonPhrase(200);
+```
 
-response
-	.status(HttpStatus.getStatusCode('Server Error'))
-	.send({
-		error: 'Server Error'
-	})
+### No wildcard imports
+We're moving away from wildcard imports in favor of selective imports.
+
+**bad** ❌
+
+```typescript
+import * as HttpStatus from 'http-status-codes';
 ```
 
-Option 2: Selective import
+**good** ✅
 
 ```typescript
-import { OK, getStatusText, getStatusCode } from 'http-status-codes'
+import { StatusCodes } from 'http-status-codes';
 
-response
-	.status(OK)
-	.send(getStatusText(OK))
+```
+These changes are __optional__ now, but may be required in a future major version so please consider making them now.
 
-response
-	.status(getStatusCode('Server Error')
-	.send('Server Error')
+
+## Proposing a new status code
+
+If you'd like to porpose a new status code, feel free to update "codes.json" with the necessary
+information and open a pull request. There is no need to modify any source code as it is all
```

Comment:
[nitpick] I'd throw in that the `README.md` is also updated automatically (I initially commented this, then read the script and realized you took care of this already, which is awesome).

Maybe something like this, though my tone is less formal:
> No need to modify source code or even this README–the `update-codes` script will handle this all for you.